.github/workflows/automated-tests.yml

```diff
@@ -24,8 +24,8 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: "1.19" # The Go version to download (if necessary) and use.
-      # - name: Proto Check
-      #   run: make proto-check
+      - name: Proto Check
```

Comment:
Merge https://github.com/cosmos/interchain-security/pull/1063 into this branch to make this check pass

Reply:
Merged